sdk/servicebus/Azure.Messaging.ServiceBus/tests/SessionRecords/ServiceBusManagementClientLiveTests/AuthenticateWithSharedKeyCredentialAsync.json

```diff
@@ -7,10 +7,10 @@
         "Authorization": "Sanitized",
         "Content-Length": "851",
         "Content-Type": "application/atom\u002Bxml",
-        "traceparent": "00-716054ab44833347a8ea439361d29ade-e25caad0d4c69345-00",
+        "traceparent": "00-6e9fe8fdd6c95046a50542def9d45550-01ea95b75dc2d64a-00",
```

Comment:
This one still seems to have been re-recorded.

Reply:
That was intentional due to the credential type change, but now that you mention it, the changes up the stack probably don't need to be reflected down this way.   Let me try reverting and see if it causes a failure.

Reply:
Looks like I don't need it.   I'll push a revert once the test runs finish.